src/gh/components/DF_cvs_exporter/code.py

```diff
@@ -0,0 +1,53 @@
+#! python3
+
+import System
+import typing
+
+import Rhino
+import Rhino.Geometry as rg
+from ghpythonlib.componentbase import executingcomponent as component
+
+import Grasshopper as gh
+from Grasshopper.Kernel import GH_RuntimeMessageLevel as RML
+
+import csv
+import os
+
+
+class CsvExporter(component):
+    def RunScript(self,
+        i_dump: bool,
+        i_export_dir: str,
+        i_results):
+        """
+            The csv-exporter component exports a list of values to a .csv file
+
+            :param i_dump: A flag indicating whether to perform the export.
+            :param i_export_dir: The directory where the CSV file will be saved.
+            :param i_values: A list of values to be exported.
+        """
+        if i_dump:
+            # Ensure the export directory exists
+            os.makedirs(i_export_dir, exist_ok=True)
+
+            # Define the CSV file path
+            file_path = os.path.join(i_export_dir, 'exported_values.csv')
```

Comment:
could we also add the possibility to the user to specify the file name (e.g. different names for different parts ) ?

Reply:
Thanks for the suggestion. Added!